### DISCUSSION TITLE
spotube feedback

### DISCUSSION THREAD

**Discussion by sospwa** [community/community#28826] [Other Feature Feedback, Questions, & Ideas]
Would be cool if it we're able to download albums and playlists. Also a tab to view downloaded music

**Comment by abbasbzu** [comment]
I agree! Downloading albums and playlists for offline listening would be a game-changer for Spotube. Being able to curate custom playlists and then access them on the go without needing an internet connection would be amazing. Plus, having a dedicated tab for downloaded music would make it easy to manage and find what you're looking for.

**Here are some additional thoughts:**

- **Granular download controls:** Allowing users to choose individual songs, albums, or entire playlists for download would offer more flexibility.
- **Background downloading:** Downloading music in the background while using other apps would be a seamless experience.
- **Storage management:** Providing options to specify download locations and manage storage space would be helpful.

While Spotube doesn't offer these features, other music streaming apps do. **Have you considered trying resso?** It allows you to download albums and playlists for offline listening, and it even has a dedicated "Downloads" tab for easy access. Resso also offers unique features like curated music discovery and social sharing.

Ultimately, the choice is yours, but I hope this information helps you find the music streaming experience that best suits your needs.

**Remember, even if Spotube doesn't have these features yet, your feedback can make a difference!** Let the developers know how much you'd appreciate the ability to download music and manage it offline. The more users who request it, the more likely they are to implement it in the future.

I hope this helps! Let me know if you have any other questions.

**Comment by Akash1134** [comment]
Hi @sospwa , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
